crates/bevy_winit/src/lib.rs

```diff
@@ -77,6 +77,29 @@ pub struct WinitPlugin;
 impl Plugin for WinitPlugin {
     fn build(&self, app: &mut App) {
         let mut event_loop_builder = EventLoopBuilder::<()>::with_user_event();
+
+        // This is needed because the X11 feature
+        // might be enabled for macos too.
```

Comment:
not just on macOS, x11 is enabled by default in Bevy on all platforms

Reply:
Thanks, new comment is hopefully more correct